chain/rust/src/builders/certificate_builder.rs

```diff
@@ -101,7 +116,9 @@ pub fn add_cert_vkeys(
         Certificate::PoolRetirement(cert) => {
             vkeys.insert(cert.ed25519_key_hash.clone());
         }
-        Certificate::RegCert(_cert) => todo!(),
+        Certificate::RegCert(_cert) => {
+            // does not require a witness
+        }
         Certificate::UnregCert(_cert) => todo!(),
```

Comment:
why todos?

Reply:
Forgot to add them here too. The ones in cert_required_wits were all tested so this should be fine since it's based on those and identical except it errors when the wits aren't vkeys. I just pushed a commit that includes these too.